CONTRIBUTING.md

```diff
@@ -13,70 +13,71 @@ Here are some ways to contribute to the project, from easiest to most difficult:
 * [Responding to issues](#responding-to-issues)
 * [Small bug fixes](#small-bug-fixes)
 * [Suggesting features](#suggesting-features)
-* [Big pull requests](#big-prs)
+* [Big pull requests](#big-pull-requests)
 
 ## Issues
 
 ### Reporting bugs
 
-If you encounter a bug, please file an issue here on GitHub, and make sure you note which library is causing the problem. If an issue you have is already reported, please add additional information or add a 👍 reaction to indicate your agreement.
+If you encounter a bug, please file an issue here on GitHub, and make sure you note which library is causing the problem. If an issue you have is already reported, please add additional information or add a "+1" comment to indicate you're affected by the issue too and this will help us prioritize the issue.
 
-While we will try to be as helpful as we can on any issue reported, please include the following to maximize the chances of a quick fix:
+While we will try to be as helpful as we can on any issue reported, please include as many details as requested in the [Bug Report issue template](https://github.com/apollographql/apollo-ios/issues/new?assignees=&labels=bug%2Cneeds+investigation&projects=&template=bug_report.yaml). Having reproducible steps or sample code for the issue will greatly speed up the time in which we can narrow down on the root cause and find a solution.
 
-1. **Intended outcome:** What you were trying to accomplish when the bug occurred, and as much code as possible related to the source of the problem.
```

Comment:
Rather than list out the actual items we want for an issue we should point to the templates as they'll be the most maintained.